edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/DependencyManager.cs

```diff
@@ -198,6 +198,13 @@ void RegisterCommonModule(ContainerBuilder builder, bool optimizeForPerformance,
                     proxy));
         }
 
+        static string GetProductInfo()
+        {
+            string version = VersionInfo.Get(Constants.VersionInfoFileName).ToString();
+            string productInfo = $"{Core.Constants.IoTEdgeProductInfoIdentifier} {version}";
```

Comment:
We should use the user agent format?

```suggestion
            string productInfo = $"{Core.Constants.IoTEdgeProductInfoIdentifier}/{version}";
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent#Syntax